sdk/tests/deqp/framework/common/tcuFuzzyImageCompare.js

```diff
@@ -125,7 +147,9 @@ var tcuTextureUtil = framework.common.tcuTextureUtil;
         var start = src.getRowPitch() * y + x * NumChannels;
         var end = start + NumChannels;
         /** @type {goog.TypedArray} */ var ptr = src.getDataPtr().subarray(start, end);
-        /** @type {Uint32Array} */ var v = new Uint32Array(ptr); //Small buffer copy
+        /** @type {Uint32Array} */ var v = new Uint32Array(ptr.buffer).subarray(
+            start * ptr.BYTES_PER_ELEMENT / 4,
+            start * ptr.BYTES_PER_ELEMENT / 4);
```

Comment:
This is a little weird. There's no point in allocating "ptr" in this case. You might as well start with src.getDataPtr().buffer.